app/interfaces/import.ts

```diff
@@ -47,14 +49,40 @@ export interface IImportFile {
   content: string[];
   withHeader?: boolean;
   delimiter?: string;
+  s3Config?: IS3Config;
+  sftpConfig?: ISftpConfig;
+  /** remote path */
+  path?: string;
+}
+
+export interface IS3Config {
+  region: string;
+  endpoint: string;
+  accessKey: string;
+  accessSecret: string;
+  bucket?: string;
+  token?: string;
+  key: string;
 }
 
+export interface ISftpConfig {
+  host: string;
+  port: number;
+  username: string;
+  password: string;
+  path: string;
+  keyFile?: string;
+  keyData?: string;
+  passPhrase?: string;
+}
```

Comment:
what are they used for?

Reply:
Keep the configuration definition the same as the nebula importer, but it is not currently used in the studio

Reply:
If they are not used in our app, you could not pre-define them now, maybe we will never use these fields, it's little redundant.